### ISSUE TITLE
Prettify adds extra lines after every comment line in CSS

### ISSUE THREAD

**Issue by sankalpdomore** [victorporof/Sublime-HTMLPrettify#492]
Whenever I use prettify for css, and my css has a comment added, it adds another line of code to my css file. Here's an example

Before using Prettify:
<img width="751" alt="before" src="https://user-images.githubusercontent.com/18334593/30956468-0be15706-a455-11e7-9d0b-14e0377fe4b2.png">

After using Prettify:
<img width="793" alt="after" src="https://user-images.githubusercontent.com/18334593/30956467-0bdfcaa8-a455-11e7-8ff9-6d3319578e25.png">

**Comment by danielhickman** [comment]
Yeah, it doesn't seem to act the same as [beautify-web/js-beautify](https://github.com/beautify-web/js-beautify).

I personally don't want to add the lines under the comments. It should be an option since it doesn't seem to occur in js-beautify, let me know if that's the case currently. Also, the one above "Why am I falling" is really annoying and obviously not intended behavior.

https://gfycat.com/popularofficialimperialeagle

**Comment by victorporof** [comment]
This behaviour doesn't originate from this plugin. But we do probably need to update the internal jsbeautify library, which had this bug.

**Comment by ollejah** [comment]
add `"preserve_newlines": false` into `"css": {...}` in `.jsbeautifyrc`

**Comment by danielhickman** [comment]
I want to preserve my existing newlines (I use two or more newlines after groups of selectors that belong to a specific element or function) but that does prevent it from adding it over and over. It doesn't prevent the addition of a newline after a comment though and both of those aren't present in the latest jsbeautify and I think an update of the library would fix that. Until then, that works as a workaround but with a trade-off.
